docs/DeveloperGuide.adoc

```diff
@@ -475,23 +475,34 @@ is rather high. Implementing alternative 2 would be more tedious on the user's s
 // tag::list[]
 === List (Mainlist, Blacklist etc) mechanism
 
-List feature is a useful feature to view persons of a certain category. In the context of this project, master list contains all the contacts of the debt collection agency.
+Multiple List feature is a useful feature to view debtors of different categories. Currently these different lists include masterlist and blacklist.
 
-We are displaying multiple lists to suit the need for the direction of the project. The lists could be viewed with the respective commands that will bring it to picture. The commands to materialize these lists are named after the lists itself. Meaning, to view the blacklist, the command typed is "blacklist" or "bl'.
+These lists could be viewed with the respective commands that will update the current displayed panel. The commands to materialize these lists are named after the lists themselves. Meaning, to view the `blacklist`, the command typed is "blacklist" or "bl'.
 
-These commands will inherit from `UndoableCommand`. Thus, these commands are supported by `undo` and `redo` command residing in the current state of Addressbook.
+These commands will inherit from `UndoableCommand`. Thus, these commands are supported by `undo` and `redo` command residing in the current state of Codii.
 
-This List mechanism is facilitated by commands which uses `Logic` interface to obtain the copy of the list that is required. As seen by the diagram 3.2.1 and 3.2.2 above, to obtain the current blacklist of the address book, "list" string is first captured by the `CommandBox`. The `CommandBox` then passes this string to the `Logic` interface for execution.
+The following sequence diagram, Figure 3.5.1, shows further details of the interaction between the user and various application components as a whole, when the blacklist command is executed:
+
+image::BlacklistCommandSequenceDiagram1.PNG[width="800"]
+Figure 3.5.1 : Sequence Diagram of How the Blacklist Command Works_
```

Comment:
rmb to put the underscore in front too (same errors below)
i.e. "_Figure 3.5.1..."

Reply:
Noted.